### DISCUSSION TITLE
Better legend & docs for seekbar ticks

### DISCUSSION THREAD

**Discussion by talmo** [talmolab/sleap#1109] [Ideas]
Credit to @lachioma:
> It would be helpful to add in the documentation a little scheme/table explaining all possible lines that can be found in the seekbar. This info does exist in the docs but is dispersed across various pages.
> 
> I did not notice that hovering over the tick you get a tooltip. This is maybe because my videos are quite long (>300k frames) and so it is very hard to place the cursor exactly on the ticks (I just tried it now). You could add a legend in the doc here: https://sleap.ai/guides/gui.html#seekbar and maybe in the GUI menu bar in Help->Seekbar legend.
This is from my personal notes, if any useful:
> - cyan: labeling suggestion + predicted (not labeled)
> - black thin: predicted (not suggested, not labeled)
> - black thick: labeled (not suggested)
> - blue filled: labeling suggestion + labeled (not predicted)

**Comment by gitttt-1234** [comment]
Closing this discussion, this is now added to the docs at [Prediction-Assisted Labeling > Understanding the Seekbar](https://docs.sleap.ai/latest/learnings/prediction-assisted-labeling/#understanding-the-seekbar)
